BLSQ/iaso
# IA-2887: Org unit picker validation status

Adding the possibility to show/hide rejected or new org units on org unit tree

Related JIRA tickets : IA-2887

## Self proofreading checklist



## Changes

- adapt bluesquare-components to be able to invalidate org unit tree queries
- adapt back-end to allow multiple validation types on org unit api
- move show types in a settings popper
- add display new / rejected switches
- adapt calls to filter org unit

## How to test

Make sure you have rejected and new org units (at a country level so you don't have to go deep)
Toggle show rejected and new org units, you should see or find those org units

## Print screen / video


https://github.com/BLSQ/iaso/assets/12494624/6cef6c34-0216-45a7-a1a7-6c53ad4af375



## Notes

- What should be the result while searching an org unit and his parent is rejected ?
- This one depends on bluesqaure-component [PR](https://github.com/BLSQ/bluesquare-components/pull/147)
